test/misc.spec.js

```diff
@@ -23,6 +24,12 @@ it('should prevent direct access to webpack compiler\'s main methods', () => {
     expect(() => compiler.webpackCompiler.watch).toThrow(/\bpublic API\b/);
 });
 
+it('should override the webpack compiler\'s outputFileSystem to a fully featured node fs', () => {
+    const compiler = createCompiler(configBasic);
+
+    expect(compiler.webpackCompiler.outputFileSystem).toMatchObject(nodeFs());
```

Comment:
I think that the test should assert that the result of nodeFs() matches require(‘fs’) and has mkdirp.
The reasoning is that this test won’t fail even if nodeFs() returned an empty object.

Reply:
👍